### DISCUSSION TITLE
Appium 8, custom WebElement - how to?

### DISCUSSION THREAD

**Discussion by DzmitryIlyin** [appium/appium#16422] [Q&A]
Hi,
I'm wondering if there's some example on how to add custom WebElement using latest Appium on Java?
I assume that it should be done by extending AppiumFieldDecorator and passing customised DefaultFieldDecorator there, but not sure where to start from.
Any help will be much appreciated :)

**Comment by KazuCocoa** [comment]
It looks like appium-java client repository is better than here.

**Reply by DzmitryIlyin** [reply]
Created the same in appium-java, please delete this one.
